### PR TITLE
Cleanup to push case bundle through after changes to operator

### PR DESCRIPTION
**Pull request by bluzarraga**

### CODE REVIEW

File: helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-audit-service:3.4.0
+    pull-repository: quay.io/opencloudio/icp-audit-service:3.4.0
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-audit-service:3.4.0
+    pull-repository: quay.io/opencloudio/icp-audit-service:3.4.0
```
Comment (bitscuit) [inline helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml]: icp-audit-service is now at 3.4.1

File: helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    pull-repository: quay.io/opencloudio/icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    pull-repository: quay.io/opencloudio/icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
```
Comment (bitscuit) [inline helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml]: icp-audit-service:3.4.1 digest is `24a220374c39652f8b031637d852995bfaec5be2601f9e4c41eb1925531f466d`

**Review by bitscuit — CHANGES_REQUESTED** [review]
I see audit service has both tag and digest in manifest.yaml, but icp-helm-repo operand only has digest. Is that correct?

File: helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml (hunk 1 above)
Comment (bluzarraga) [reply helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml]: Good catch, I'll add the tag values for helm-repo operand as well.

**Review by bitscuit — APPROVED** [review]
/lgtm

**Pull request merged**